README.md

```diff
@@ -1 +1,78 @@
-# ocw-course-hugo-starter
\ No newline at end of file
+# ocw-course-hugo-starter
+
+A starter template for a Hugo site that uses [`ocw-course-hugo-theme`](https://github.com/mitodl/ocw-course-hugo-theme)
+
+![image](https://user-images.githubusercontent.com/12089658/102410491-9b7e1680-3fbe-11eb-9746-e5ae996add40.png)
+
+## Prerequisites
+
+ - NodeJS
+ - Hugo
+
+## Building JS & CSS
+
+To build the JS and CSS for the theme, make sure you have the above prerequisites installed and run the following:
+
+```sh
+./build.sh
+```
+
+## Running the site
+
+To preview your site, run the following command:
+
+```sh
+hugo server
+```
+
+Hugo will watch your site source and refresh the page as you make changes to the site content.
+
+## Adding content
+
+Course level metadata for your course should be placed in the JSON format under the `data/courses` folder with a filename matching the course's ID.  For example: `data/courses/18-01-single-variable-calculus-fall-2006.json`.
```

Comment:
Does any specific metadata need to be in this file or can it be whatever the user wants?

Reply:
this has to be the json file output from ocw-to-hugo

Reply:
This is why I put a link to the theme readme at the bottom which has a breakdown of all of the expected properties of the JSON file.  I remember writing this now debating where to put that exactly but in hindsight I think it belongs here.